app.js

```diff
@@ -102,3 +102,62 @@ document.querySelector('.displayed-books').addEventListener('click', (e) => {
   Store.deleteBook(bookToRemove);
   
 });
+
+const currentDate = document.querySelector('.current-date');
+
+const updateClock = () => {
+  let today = new Date();
+  let month = today.getMonth();
+  let year = today.getFullYear();
+  let date = today.getDate();
+  let hours = today.getHours();
+  let minutes = today.getMinutes();
+  let seconds = today.getSeconds();
+  const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+  currentDate.innerHTML = `
+    <p>${months[month]} ${date}th ${year},  ${hours}:${minutes}:${seconds}</p>
+  `;
+}
+
+const initClock = () => {
+  updateClock();
+  window.setInterval('updateClock()', 1);
+}
+
+document.addEventListener('DOMContentLoaded', initClock);
+
+const navLinks = document.querySelectorAll('.item');
+const firstSection = document.querySelector('.first-section');
+const secondSection = document.querySelector('.second-section');
+const thirdSection = document.querySelector('.third-section');
+
+navLinks.forEach((link, index) => {
+  if(index === 0) {
+    link.addEventListener('click', () => {
+      link.classList.add('link-color');
+      link[1].classList.remove('link-color');
+      link[2].classList.remove('link-color');
```

Comment:
- [ ] When I navigate to the "Add New" and "Contact" sections I cannot navigate back to the "List"

  ![image](https://user-images.githubusercontent.com/81468675/212115660-a540bb97-fc57-46c1-b390-84c593f30c18.png)

  You can fix this by changing `link` to `navLinks`

  ```
  navLinks[1].classList.remove('link-color');
  navLinks[2].classList.remove('link-color');
  ```

Reply:
Thanks. That is now fixed.